pkg/apiserver/schedule/schedule.go

```diff
@@ -579,6 +581,31 @@ func (s *Service) getScheduleDetail(c *gin.Context) {
 		return
 	}
 
+	UIDList := make([]string, 0)
+	kind, ok := v1alpha1.AllScheduleItemKinds()[string(schedule.Spec.Type)]
+	if !ok {
+		c.Status(http.StatusInternalServerError)
+		_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
+		return
+	}
+	list := kind.ChaosList.DeepCopyObject()
```

Comment:
Why not using `utils.ActiveLister` directlly?

Reply:
Because the existing components can solve the problem, no more components are introduced.

Reply:
I mean, the code is quite the same as `utils.ActiveLister`, you can just reuse it,

Reply:
I just tried the "ListActiveJobs" in utils.ActiveLister, and its return value is runtime.Object, so it's not as convenient to write it again. . . .